### PR TITLE
Support answers-format option in install

### PR DESCRIPTION
**Pull request by sub-mod**
support answers-format option in install
so that we can define the format of answers.conf.sample file.

default = 'ini'
Valid formats are 'ini', 'json', 'xml', 'yaml'.

example:
atomicapp -v install --answers-format="json"  submod/guestbookphp-app

Required for https://github.com/cockpit-project/cockpit/pull/2474

### CODE REVIEW

File: atomicapp/cli/main.py [hunk 1]
```diff
@@ -49,6 +49,7 @@ def set_arguments(self):
         parser_install = subparsers.add_parser("install")
 
         parser_install.add_argument("--no-deps", dest="nodeps", default=False, action="store_true", help="Skip pulling dependencies of the app")
+        parser_install.add_argument("--answers-format", dest="answers_format", default=ANSWERS_FILE_SAMPLE_FORMAT, help="The format for the answers.conf.sample file.Default is 'ini', Valid formats are 'ini', 'json', 'xml', 'yaml'.")
```
Comment (vpavlin) [inline atomicapp/cli/main.py]: Please move this option to global parser (i.e. `atomicapp --answers-format ...` not `atomicapp install --answers-format` as `answers.conf.sample` is generated also during `run` if there is no answers file found.

**Comment by sub-mod** [comment]
@goern kindly review

**Comment by aweiteka** [comment]
LGTM

File: atomicapp/install.py [hunk 2]
```diff
@@ -147,7 +147,7 @@ def install(self):
 
         printAnswerFile(json.dumps(answerContent))
         printStatus("Install Successful.")
-        return values
+        return None
```
Comment (goern) [inline atomicapp/install.py]: does not return anything any longer? @vpavlin @sub-mod

Comment (sub-mod) [inline atomicapp/install.py]: @goern run() also returns None.
The funny thing is i need None for the testcases to work.

**Comment by sub-mod** [comment]
@goern thanks

**Comment by stefwalter** [comment]
When will this be included in a release? And included in nulecule images?

**Comment by goern** [comment]
This is always be in a :dev image, as that corresponds to the master branch. We are currently planning the CDK release and will align Atomic App base image releases with that... I hope we can finish that tomorrow.

**Pull request merged**